feat(core): update prettier to v2.1.2

mehrad-rafigh opened the pull request:
## Current Behavior
prettier is at v2.0.4

## Expected Behavior
prettier is at v2.1.2

## Related Issues

vsavkin commented on packages/workspace/src/migrations/update-10-3-0/update-10-3-0.ts:
```diff
@@ -0,0 +1,11 @@
+import { join } from 'path';
+import { chain } from '@angular-devkit/schematics';
+
+const updatePrettierPackage = updatePackagesInPackageJson(
```
Can you rename it to updatePackages? Other packages can be updated there as well

vsavkin commented:
Thank you for your PR!

Could you also update prettier in the root package.json in the Nx repo itself?

mehrad-rafigh commented:
Hi @vsavkin 

> Could you also update prettier in the root package.json in the Nx repo itself?
I updated prettier to 2.1.2 in the package.json of the repo itself and renamed the rule to `updatePackages`

mehrad-rafigh commented:
Hi @vsavkin Any chance this can get merged?

vsavkin commented:
Sorry about the delay. Merged it here: https://github.com/nrwl/nx/commit/e3f318cde33b3dc68c0374af77271d66f6a01af4

Thank you!
